lib/period.spec.js

```diff
@@ -243,6 +243,54 @@ describe('periods', () => {
     done();
   });
 
+  it('should allow to get proof from period with prevPeriod.', (done) => {
+    const prevTx = '0x7777777777777777777777777777777777777777777777777777777777777777';
+    const value = 99000000;
+    const color = 1337;
+    const transfer = Tx.transfer(
+      [new Input(new Outpoint(prevTx, 0))],
+      [new Output(value / 2, ADDR, color), new Output(value / 2, ADDR, color)],
+    );
+    transfer.sign([PRIV]);
+
+    const blocks = [];
+    let block;
+    for (let i = 0; i < 31; i++) {
+      if (i % 2) {
+        block = new Block(i).addTx(Tx.deposit(i, value, ADDR, color));
+        blocks.push(block);
+      } else {
+        blocks.push(new Block(i));
+      }
+    }
+    block = new Block(31).addTx(transfer);
+    blocks.push(block);
+
+    const period = new Period("'0x8b04de057fe524a3118eb7c8e14a2e55323c67fd7b6080583d1047b700b2d674'", blocks);
+    period.setValidatorData(slotId, ADDR);
+    const proof = period.proof(transfer);
+    expect(proof).to.eql([
```

Comment:
this test is not checking that prevHash is included in the proof, no?

How about that:
1. period = new Period(prevHash, blocks)
2. const proof1 = period.proof(transfer)
3 period = new Period(someOtherPrevHash, blocks)
4. const proof2 = period.proof(transfer)
5. expect(proof1).not.equal(proof2)

Reply:
or better check for prevHash inclusion?